### PR TITLE
enclosure js code into anonymous function

### PR DESCRIPTION
**Pull request by jbcr**

### CODE REVIEW

File: src/Resources/views/themes/admin/ezdataflow/Dashboard/main.html.twig [hunk 1]
```diff
@@ -34,6 +34,9 @@
     </div>
 
     <script>
+
+        (function ($) {
+            $(document).ready(function ($) {
         $('#ez-modal--history-details').modal({keyboard: false, show: false});
```
Comment (mfarrica-cr) [inline src/Resources/views/themes/admin/ezdataflow/Dashboard/main.html.twig]: Could you reindent this block too ?

File: src/Resources/views/themes/admin/ezdataflow/parts/tab/job_list.html.twig [hunk 2]
```diff
@@ -54,6 +54,9 @@
 </div>
 
 <script>
+
+    (function ($) {
+        $(document).ready(function ($) {
     // Manage ajax pagination
```
Comment (mfarrica-cr) [inline src/Resources/views/themes/admin/ezdataflow/parts/tab/job_list.html.twig]: Same here ;)

File: src/Resources/views/themes/admin/ezdataflow/parts/tab/schedule_list.html.twig [hunk 3]
```diff
@@ -71,6 +71,9 @@
 </div>
 
 <script>
+
+    (function ($) {
+        $(document).ready(function ($) {
     // Manage ajax pagination
```
Comment (mfarrica-cr) [inline src/Resources/views/themes/admin/ezdataflow/parts/tab/schedule_list.html.twig]: Here too :P

**Review by mfarrica-cr — APPROVED** [review]
With a reindent for commented blocks.

**Comment by jbcr** [comment]
This change has been applied on 2.0.x branch too

**Pull request merged**